events/tests/test_organization_api.py

```diff
@@ -272,6 +270,31 @@ def test_admin_user_add_users_to_new_organization(
     )
 
 
+@pytest.mark.django_db
+def test_admin_user_add_registration_admin_users_to_new_organization(
+    api_client, organization, data_source, user, user2, user_api_client
+):
+    organization.admin_users.add(user)
+    api_client.force_authenticate(user)
+    origin_id = "test_organization2"
+    payload = {
+        "data_source": data_source.id,
+        "origin_id": origin_id,
+        "id": f"{data_source.id}:{origin_id}",
+        "name": organization_name,
+        "admin_users": [user.username, user2.username],
+        "registration_admin_users": [user2.username],
+    }
+
+    response = assert_create_organization(user_api_client, payload)
+    assert set(payload["admin_users"]) == set(
+        [i["username"] for i in response.data["admin_users"]]
+    )
+    assert set(payload["registration_admin_users"]) == set(
+        [i["username"] for i in response.data["registration_admin_users"]]
+    )
+
+
```

Comment:
I think it is better when these tests also check the database just to be sure, as is done in the other new test `test_admin_user_update_organization_registration_admin_users()`.

Reply:
Changed as suggested now